spec/Section 6 -- Execution.md

```diff
@@ -236,12 +236,12 @@ connectivity.
 **Delivery Agnostic**
 
 GraphQL subscriptions do not require any specific serialization format or
-transport mechanism. Subscriptions specifies algorithms for the creation of a
-stream, the content of each payload on that stream, and the closing of that
-stream. There are intentionally no specifications for message acknowledgement,
-buffering, resend requests, or any other quality of service (QoS) details.
-Message serialization, transport mechanisms, and quality of service details
-should be chosen by the implementing service.
+transport mechanism. GraphQL specifies algorithms for the creation of a
```

Comment:
This now sounds wrong and introduces a new term `subscription stream`. Usually, we talk about the response stream as the thing that is produced and the source stream or event stream as the thing we subscribe to. I think the sense here also is that a subscription is in essence something you wrap around the standard executor. 

So, `Subscriptions specifies algorithms for the creation of a stream` or `GraphQL Subscriptions specifies algorithms for the creation of a stream` or if we want to be more explicit we could say `Subscriptions specifies algorithms for the creation of a response stream`

I will reread the whole section and reason a bit about it.

Reply:
Reflected a bit on this with @benjie ... 

his suggestion goes in the direction of 

`GraphQL specifies algorithms for the creation of a stream used for a subscription` which I think sounds better.

Reply:
Improved wording available in https://github.com/graphql/graphql-spec/pull/959; I suggest removing this rewording from this PR so that it can be discussed separately.

Reply:
ok

Reply:
reverted this piece except fixing grammar (Subscriptions specifies -> Subscription specifies)